conda-recipe/meta.yaml

```diff
@@ -43,6 +43,7 @@ test:
         - pandas
         - scipy
         - scikit-learn
+        - threadpoolctl
```

Comment:
It should be like a temporary solution. because in runtime we shouldn't require this dependency from the user.

Reply:
yes. But this is not a runtime but test env. this will have no impact on users

Reply:
ok